### ISSUE TITLE
a11y: Individual project release management table

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6080]
Minor accessibility improvements can be made to `/manage/project/{project_name}/release/{release}`:

* The asset `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the `<h2>` above
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node
* The option dropdown buttons should have `type=button` to indicate that they don't reset or resubmit any form data

**Comment by woodruffw** [comment]
In the hashes modal for each release asset:

* The hashes `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the <h2> above
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node
* The clipboard copy buttons should have `type=button` to indicate that they don't reset or resubmit any form data
